lib/src/sdk.ts

```diff
@@ -24,9 +24,13 @@ export type SwapInfo = {
   toAccountId: string;
   fromAmount: BigNumber;
   feeStrategy: FeeStrategy;
+  maxFeePerGas?: BigNumber;
```

Comment:
I think it is best to wrap up all those "custom fees parameters" together, like a `customFees` param.

It seems also that there are some missing parameter that you required in your code (like `feePerByte`).